packages/apollo/src/commands/service/check.ts

```diff
@@ -351,9 +347,9 @@ export default class ServiceCheck extends ProjectCommand {
                 const {
                   compositionValidationResult,
                   checkSchemaResult
-                } = await project.engine.checkPartialSchema({
-                  id: graphName,
-                  graphVariant: tag,
+                } = await serviceProject.engine.checkPartialSchema({
+                  id: graphID!,
```

Comment:
I think we should assert the existence of `graphID` and `graphVariant` and deal with that above instead of using non-null assertions here?  There are a few more of these assertions below, I can come back and mark them based on your thoughts here.